src/components/function-summary/result.tsx

```diff
@@ -15,6 +15,15 @@ export const FunctionSummaryResult = ({ result, txStatus }: FunctionSummaryResul
   if (!result) return null;
   const { success, type, value } = cvToJSON(hexToCV(result.hex));
   const hasType = !type?.includes('UnknownType');
+
+  const getReprValue = (item: any) => {
+    let reprValue = item.value ?? 'none';
+    if (item.type.includes('list')) {
+      reprValue = item.value.map((listEntry: any) => listEntry.value).join(', ');
+    }
+    return typeof reprValue === 'object' ? JSON.stringify(reprValue) : reprValue.toString();
```

Comment:
Please try to avoid calling `toString` for the list case as it's already a string. A similar implementation like existing one should do the trick.